### PR TITLE
fuchsis_input

### PR DESCRIPTION
**Pull request by TFuchsi**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -7,17 +7,27 @@ readme = "README.md"
 packages = [{include = "vector_intro"}]
 
 [tool.poetry.dependencies]
-python = ">=3.10,<3.13"
-sentence-transformers = "^2.2.2"
```
Comment (georgms) [inline pyproject.toml:11]: ❓ We are using `sentence-transformers`, `pandas`, `matplotlib`, `scikit-learn`, `jupyter` and `rise`. Why did you remove them?

Comment (TFuchsi) [reply pyproject.toml:11]: I didn't remove them, I just reorganised them.
